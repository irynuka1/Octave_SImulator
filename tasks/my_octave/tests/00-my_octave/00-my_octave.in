L 4 3
-10 3 2
-10 -5 9
-1 -9 -4
2 7 -6
L 2 3
-1 9 -7
1 -6 -10
T 1
M 0 1
D 2
P 2
Q