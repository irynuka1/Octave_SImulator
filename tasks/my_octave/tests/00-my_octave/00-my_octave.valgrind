==12473== Memcheck, a memory error detector
==12473== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==12473== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==12473== Command: /home/irynuka1/Desktop/Programs/Tema2_1/check_my_octave_v1.1/check_my_octave/my_octave
==12473== Parent PID: 12472
==12473== 
==12473== 
==12473== HEAP SUMMARY:
==12473==     in use at exit: 0 bytes in 0 blocks
==12473==   total heap usage: 32 allocs, 32 frees, 8,608 bytes allocated
==12473== 
==12473== All heap blocks were freed -- no leaks are possible
==12473== 
==12473== For lists of detected and suppressed errors, rerun with: -s
==12473== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
