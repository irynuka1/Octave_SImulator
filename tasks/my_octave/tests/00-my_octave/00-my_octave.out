4 2
23 9959 
9909 9937 
9955 93 
103 20 
